### PR TITLE
Update querying.md

### PR DESCRIPTION
**Pull request by mcclowes**
# Description

- Added qualifying info about restrictions regarding querying arrays
- Tidied up existing documentation

### CODE REVIEW

File: docs/using-the-api/querying.md [hunk 1]
```diff
@@ -52,12 +55,13 @@ Our `GET /{dataType}` endpoints typically return an array of items of that given
 
 ## Example queries
 
-Note that characters (<, >) are url-encoded.
+**Note:** some characters are url-encoded:
+- < = `%3c`
+- > = `%3e`
```
Comment (pzaichkina) [inline docs/using-the-api/querying.md:60]: have a look at the preview for - > = `%3e` - something is not rendering correctly with this syntax. 
image

File: docs/using-the-api/querying.md [hunk 2]
```diff
@@ -268,4 +286,16 @@ var response = client.Execute(request);
 var info = response.Data;
 ```
 </TabItem>
-</Tabs>
\ No newline at end of file
+</Tabs>
```
Comment (pzaichkina) [inline docs/using-the-api/querying.md:289]:
```suggestion
</Tabs>  


```

Comment (pzaichkina) [reply docs/using-the-api/querying.md:289]: the markdown is currently not recognized for the new section
image

File: docs/using-the-api/querying.md [hunk 3]
```diff
@@ -268,4 +286,16 @@ var response = client.Execute(request);
...
+  
+## Queries that won't work
+
+Although you can query properties of objects, you can't query arrays.
```
Comment (pzaichkina) [inline docs/using-the-api/querying.md:293]:
```suggestion
Although you can query properties of objects, you can't query arrays.  For example:
```

File: docs/using-the-api/querying.md [hunk 4]
```diff
@@ -268,4 +286,16 @@ var response = client.Execute(request);
...
 </TabItem>
-</Tabs>
\ No newline at end of file
+</Tabs>
+  
+## Queries that won't work
+
+Although you can query properties of objects, you can't query arrays.  
+
+✅ E.g. objects - Invoices > `customerRef.id`
```
Comment (pzaichkina) [inline docs/using-the-api/querying.md:295]:
```suggestion
✅ Objects: Invoices > `customerRef.id`
```

File: docs/using-the-api/querying.md [hunk 5]
```diff
@@ -268,4 +286,16 @@ var response = client.Execute(request);
...
+  
+`GET /invoices?page=1&pageSize=100&query=customerRef.id%3Def6f54c1-eb45-4956-b8cd-1be82ad665f2`
+  
+❌ E.g. arrrays - Invoices > `lineItems`
```
Comment (pzaichkina) [inline docs/using-the-api/querying.md:299]:
```suggestion
❌ Arrays: Invoices > `lineItems`
```

**Review by pzaichkina — CHANGES_REQUESTED** [review]

File: docs/using-the-api/querying.md (hunk 1 above)
Comment (mcclowes) [reply docs/using-the-api/querying.md]: Good spot, fixed it

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**